src/sentry/utils/committers.py

```diff
@@ -254,7 +246,11 @@ def get_serialized_event_file_committers(project, event, frame_limit=25):
 
     for committer in committers:
         commit_ids = [commit.id for (commit, _) in committer["commits"]]
-        committer["commits"] = [serialized_commits_by_id[commit_id] for commit_id in commit_ids]
+        commits_result = [serialized_commits_by_id[commit_id] for commit_id in commit_ids]
```

Comment:
If commits were indexed by the `key` would you be able to skip de-duplication? While we don't have constraints on commit.key, it *should* be uniqueish and is the 'id' we use when serializing commits.

Reply:
@markstory when I checked on redash, there was only one commit, but we ended up with duplicates. I'm honestly not sure where and how the commits are being duplicated in the code. I suspect it within this function: https://github.com/getsentry/sentry/blob/c9cb310884029f685eec6b56dd8247616f83294a/src/sentry/utils/committers.py#L169 .